runtime/src/installed_scheduler_pool.rs

```diff
@@ -518,10 +513,11 @@ mod tests {
     fn test_scheduler_pause() {
         solana_logger::setup();
 
-        let bank = Arc::new(crate::bank::tests::create_simple_test_bank(42));
+        let bank = &Arc::new(crate::bank::tests::create_simple_test_bank(42));
```

Comment:
why all these changes to use an `&Arc<..>` instead of owned Arc?

Reply:
no strong reason other than to minimize diff. ;)

Reply:
well, thanks for pointing out. i've found way of further reducing the diff: 5df66e0aaa3